tests/MailAlterTest.php

```diff
@@ -73,6 +73,175 @@ public function testNoReplacingWithExplicitReplyTo() {
     $this->assertEqual('explicit-reply-to@other.com', $message['headers']['Reply-To']);
   }
 
+  /**
+   * Return-Path is set.
+   */
+  public function testSetReturnPath() {
+    $this->setConfig(['site_mail_return_path' => 'bounce@example.com']);
+    $message['from'] = 'some@example.com';
+    email_check_mail_alter($message);
+    $this->assertEqual('some@example.com', $message['from']);
+    $this->assertEqual('bounce@example.com', $message['headers']['Return-Path']);
+  }
+
+  /**
+   * Return-Path is only set when in mail domain.
+   * Default is the site_mail.
```

Comment:
Function doc summary should be 1 line (optionally) followed by a blank line and the rest of the doc. (phpcs)

There is many more instances of that in here.